### PR TITLE
037 implement eeli wbaccinelli

### PR DESCRIPTION
**Pull request by wbaccinelli**
Implemented EELI 

Fixes: #37

### CODE REVIEW

**Comment by DaniBodor** [comment]
I got an error when switching from a lowpass to a bandstop filter in the dropdown menu. If it's just that filter that's not working, maybe remove it for now.

> Traceback (most recent call last):
>   File "/home/dbodor/git/EIT-ALIVE/eit_dash/eit_dash/callbacks/preprocessing_callbacks.py", line 570, in enable_apply_button
>     and co_low > 0
> TypeError: '>' not supported between instances of 'NoneType' and 'int'

**Comment by DaniBodor** [comment]
PRE-PROCESSING page
- Really nice that the preprocessing page now also has info on the selected regions in the summary section.
- The "Data filtered" box under Results is not removable and only 1 filter can be stored. I think it's overwritten when a new one is stored. It's also not associated to any data, but is just the filter itself, is that on purpose? If so, it should be called "Filter" rather than "Data filtered"

ANALYZE page
- When I click "Apply EELI", I get an empty dropdown box and no options to do anything.

**Comment by wbaccinelli** [comment]
> I got an error when switching from a lowpass to a bandstop filter in the dropdown menu. If it's just that filter that's not working, maybe remove it for now? It doesn't stop me from proceeding though.
> 
> > Traceback (most recent call last):
> > File "/home/dbodor/git/EIT-ALIVE/eit_dash/eit_dash/callbacks/preprocessing_callbacks.py", line 570, in enable_apply_button
> > and co_low > 0
> > TypeError: '>' not supported between instances of 'NoneType' and 'int'

fixed [here](https://github.com/EIT-ALIVE/eit_dash/pull/42/commits/96eae5b379d82969a7262d54dc42458a495aad88)

**Comment by wbaccinelli** [comment]
> PRE-PROCESSING page
> 
> * Really nice that the preprocessing page now also has info on the selected regions in the summary section.
> * The "Data filtered" box under Results is not removable and only 1 filter can be stored. I think it's overwritten when a new one is stored. It's also not associated to any data, but is just the filter itself, is that on purpose? If so, it should be called "Filter" rather than "Data filtered"

Yes, this is on purpose and it filters all the periods. Only one type of filter can be applied at the moment and if a new filter is applied, it will override the previous results. I used "Data filtered", because the result is that the data have been filtered with the parameters reported, but I don't mind changing it,

> ANALYZE page
> 
> * When I click "Apply EELI", I get an empty dropdown box and no options to do anything.
> 
> Error message (when entering ANALYZE):
> 
> > Traceback (most recent call last):
> > File "/home/dbodor/git/EIT-ALIVE/eit_dash/eit_dash/callbacks/analyze_callbacks.py", line 49, in page_setup
> > filter_params = period.get_data().continuous_data.data["global_impedance_filtered"].parameters
> > KeyError: 'global_impedance_filtered'
Did you get this after applying the filter? This makes me think that I have to disable the option if no filtered data are available, or I need to use the raw data if the filtered data are not there.

File: docs/user_manual.md
Comment (DaniBodor) [inline docs/user_manual.md:1]: Maybe add a note at the very top asking people to open issues if they encounter anything?

File: docs/user_manual.md [hunk 1]
```diff
@@ -72,3 +72,12 @@ the preview will update. When you are happy with a particular setting/selection,
 <kbd>
 <img src=images/preprocessing_filter.png width="800px" style="border: 1px solid black">
 </kbd>
```
Comment (DaniBodor) [inline docs/user_manual.md:74]:
```suggestion
<kbd>
<img src=images/preprocessing_filter.png width="800px" style="border: 1px solid black">
</kbd>

When you have selected your period and filters, ensure they are both listed on the right side under "Results", and then click on **`ANALYZE`** at the top of the page.
```

**Review by DaniBodor — APPROVED** [review]
Great job!

I added 2 minor comments about the user manual. Also I noticed a typo that I made in that document at line 44, where Click is spelled with a capital and shouldn't be. Could you fix that here as well, please.

**Pull request merged**